### ISSUE TITLE
rhadadi@Rooholahs-MBP ~/.o/t/oh-my-fish> curl -L https://github.com/oh-my-fish/oh-my-fish/raw/master/bin/install | fish    % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current                                  Dload  Upload   Total   Spent    Left  Speed 100   140  100   140    0     0    951      0 --:--:-- --:--:-- --:--:--   952 100 16622  100 16622    0     0  55048      0 --:--:-- --:--:-- --:--:-- 55048 Existing installation detected at /Users/rhadadi/.local/share/omf Would you like to remove the existing installation? (y/N): <W> fish: No TTY for interactive shell (tcgetpgrp failed) setpgid: Inappropriate ioctl for device rhadadi@Rooholahs-MBP ~/.o/t/oh-my-fish>

### ISSUE THREAD

**Issue by Rhadadi** [fish-shell/fish-shell#4173]

**Comment by Rhadadi** [comment]
I am trying to install _omf_ using
`curl -L https://github.com/oh-my-fish/oh-my-fish/raw/master/bin/install | fish
`
but receiving an error and cannot find `fish.config` in my machine. 
Any advice?
Thanks

```
xxx@xxx-MBP ~/.o/t/oh-my-fish> curl -L https://github.com/oh-my-fish/oh-my-fish/raw/master/bin/install | fish

  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   140  100   140    0     0    951      0 --:--:-- --:--:-- --:--:--   952
100 16622  100 16622    0     0  55048      0 --:--:-- --:--:-- --:--:-- 55048
Existing installation detected at /Users/xxx/.local/share/omf
Would you like to remove the existing installation? (y/N): <W> fish: No TTY for interactive shell (tcgetpgrp failed)
setpgid: Inappropriate ioctl for device
xxxx@xxx-MBP ~/.o/t/oh-my-fish>omf update
fish: Unknown command 'omf'
```

**Comment by krader1961** [comment]
@Rhadadi, I'm curious what the problem was. It might help someone in the future if you add a comment explaining how you fixed this problem.

**Comment by Rhadadi** [comment]
@krader1961 
It was asking me if I Would you like to remove the existing installation. But it did not give me the option to select yes and was giving me the error. I  deleted the omf directory manually and the re-install the omf using the curl. It was resolved.

**Comment by krader1961** [comment]
Okay, that probably means you're using a fish older than version 2.6.0. We fixed a couple of bugs recently involving the management of process groups the tty ownership. It's always a good idea when opening an issue to include information about your environment such as the version of the relevant programs. Thanks for updating the issue.

**Comment by Rhadadi** [comment]
I installed the latest fish version (2.6)and that error happened when I was following some tutorials to install omf theme. Thanks for the follow up and for the advice. 

	





On Fri, Jun 30, 2017 at 11:02 PM -0400, "Kurtis Rader" <notifications@github.com> wrote:












Okay, that probably means you're using a fish older than version 2.6.0. We fixed a couple of bugs recently involving the management of process groups the tty ownership. It's always a good idea when opening an issue to include information about your environment such as the version of the relevant programs. Thanks for updating the issue.
